gslab_scons/README.md

```diff
@@ -45,17 +45,17 @@ github_token: <your token here>
 
 ## Provenance
 
-Making a release using this tool will automatically call `make_provenance()` from the module `provenance`create a `provenance` file (for more details, please see our [RA manual](https://github.com/gslab-econ/ra-manual/wiki/Data-Storage)). 
+Making a release using this tool will automatically call `make_provenance()` from the module `provenance` to create a `provenance` file (for more details, please see our [RA manual](https://github.com/gslab-econ/ra-manual/wiki/Data-Storage)). 
 
-While calling the `release` module from the command line, one can specify the `make_provenance()` argument by adding `detail_limit=<###>` (default to 500).
+While calling the `release` module from the command line, one can specify the `detail_limit` argument to`make_provenance()` by adding `detail_limit=<###>` (default to 500).
```

Comment:
While calling the `release` module from the command line, one can specify the `detail_limit` argument of `make_provenance()` by adding `detail_limit=<###>` (default to 500).